src/unit_tests/wazuh_db/test_wdb_global.c

```diff
@@ -957,6 +957,302 @@ void test_wdb_global_get_agent_max_group_priority_success(void **state)
     __real_cJSON_Delete(j_result);
 }
 
+/* Tests wdb_global_sync_agent_groups_get */
+
+void test_wdb_global_sync_agent_groups_get_invalid_condition(void **state)
+{
+    wdbc_result result = WDBC_OK;
+    test_struct_t *data  = (test_struct_t *)*state;
+    wdb_groups_sync_condition_t condition = 3;
+    int last_agent_id = 0;
+    bool set_synced = true;
+    bool get_hash = true;
```

Comment:
I created the variable for clarification instead of putting a "0" when is required. If we consider this change, wouldn't we need to do that with variables like "return" as well?
Also, I considered that since this is not an object that needs to be dynamically allocated like "wdb" or "output" we could repeat that variable on every test when is needed.